demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -85,13 +91,35 @@ def get_docker_python_path(docker_prefix: str) -> List[str]:
             docker_python_path.append(
                 f"{docker_prefix}/{path.relative_to(CONTENT_PATH.absolute())}"
             )
+
     if (
-        f"{docker_prefix}/Packs/Base/Scripts/CommonServerPython"
+        f"{docker_prefix}/Packs/Base/Scripts/CommonServerPython"  # CommonServerPython does not exist locally
```

Comment:
why it means it don't exist locally?

Reply:
if line 92 produces ValueError silently, then the `CommonServerPython` is likely does not exist.